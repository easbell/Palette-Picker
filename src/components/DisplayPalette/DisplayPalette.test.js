import React from 'react'
import { DisplayPalette } from './DisplayPalette'
import { shallow } from 'enzyme'
// Enzyme.configure({ adapter: new Adapter() })

describe('DisplayPalette', () => {
  let wrapper
  let mockPalette = {
    id: 4,
    palette_name: 'my palette',
    color_1: 'blue',
    color_2: 'pink',
    color_3: 'yellow',
    color_4: 'green',
    color_5: 'red',
    project_id: 7
  }

  it('should match the snapshot', () => {
    wrapper = shallow(<DisplayPalette {...mockPalette} />)
    expect(wrapper).toMatchSnapshot()
  })
})