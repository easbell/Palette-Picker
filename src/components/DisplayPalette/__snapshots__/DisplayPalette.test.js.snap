// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayPalette should match the snapshot 1`] = `
<div
  className="palette-card"
>
  <h3
    className="palette-name"
  >
    my palette
  </h3>
  <div
    className="display-colors"
  >
    <div
      className="display-color"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className="display-color"
      style={
        Object {
          "backgroundColor": "pink",
        }
      }
    />
    <div
      className="display-color"
      style={
        Object {
          "backgroundColor": "yellow",
        }
      }
    />
    <div
      className="display-color"
      style={
        Object {
          "backgroundColor": "green",
        }
      }
    />
    <div
      className="display-color"
      style={
        Object {
          "backgroundColor": "red",
        }
      }
    />
  </div>
</div>
`;
